units_cpp/src/unitsTest/headers/units/all_units.h

```diff
@@ -18,12 +18,14 @@
 #include <units/frequency_units.h>
 #include <units/acceleration_units.h>
 #include <units/density_units.h>
+#include <units/pressure_units.h>
 
 
 #ifndef UNIT_TESTS_ALL_UNITS_H_
 #define UNIT_TESTS_ALL_UNITS_H_
 
 #define LENGTH_UNITS() \
+
```

Comment:
This line drop will make the code not to compile.

Try to edit the units_cpp.stg file first before generating

Reply:
I tried fixing units_cpp.stg but couldn't find what causes the newline.
I edited all_units.h manually for now.

Can we merge this?